lib/modules/manager/api.ts

```diff
@@ -19,6 +19,7 @@ import * as cloudbuild from './cloudbuild';
 import * as cocoapods from './cocoapods';
 import * as composer from './composer';
 import * as conan from './conan';
+import * as cpanfile from './cpanfile';
```

Comment:
import * as cpanfile from './cpanfile

- [ ] `[](url)`
```suggestion

- import * as cpanfile from ['./cpanfile';](/)

```